docs/docs/Components/components-agents.md

```diff
@@ -1,7 +1,35 @@
-# Agents
+---
+title: Agents
+sidebar_position: 12
+slug: /components-agents
+---
 
 Agent components are used to define the behavior and capabilities of AI agents in your flow. Agents can interact with APIs, databases, and other services, but can also use LLMs as a reasoning engine to decide which course to take in your flow.
 
+## Agent component {#agent-component}
+
+This component creates an agent that can use tools to answer questions and perform tasks based on given instructions.
+
+For more information on this component, see the [tool calling agent documentation](/agents-tool-calling-agent-component).
+
+### Parameters
```

Comment:
Avoid empty headings. In other words, headings that aren't followed by text. Remove every instance of the "Parameters" heading from this file.